src/cdomains/apronDomain.ml

```diff
@@ -263,7 +263,7 @@ struct
         that come from the expression we wish to assert. *)
         A.meet_lincons_array Man.mgr d ea
       | None -> d
-    with Invalid_argument "cil_exp_to_lexp" -> d
+    with Invalid_argument _ -> d
```

Comment:
Did you remove these strings just because the OCaml compiler warned about them or was there an actual reason?

Also, is `ApronDomain` actually used for anything here? If not and `OctApronDomain` is used instead, then I guess there's no reason to change (or even compile) this file at all.

Reply:
Actually, that was my suggestion here. I think we should aim to treat warnings as errors.

Reply:
The sensible solution is then to make a new `exception` type that `cil_exp_to_lexp` raises and catch that specifically. Catching arbitrary `Invalid_argument` is dangerous, because many standard library functions also raise those, so if there's accidentally such coding error inside `cil_exp_to_lexp`, then it would be swallowed here.